metadat/developer/Nintendo - Game Boy Advance.dat

```diff
@@ -12187,15 +12187,15 @@ game (
 )
 
 game (
-	name "Pocket Monsters - Fire Red (Japan)"
+	name "Pocket Monsters - FireRed (Japan)"

```

Comment:
I mad this change because everywhere else in the libretro-database already has `FireRed`. Where can I push the changes to No-intro?